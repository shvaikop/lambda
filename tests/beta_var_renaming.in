(\x.(\y.x y) (\x.(\y.x y) z))