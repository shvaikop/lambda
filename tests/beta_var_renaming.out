Up to date
Welcome to Lambda Calculus Interpreter
\x.x \x.x z
